resources/views/waitingrooms/show.blade.php

```diff
@@ -9,26 +9,39 @@
 
     <div class="container">
         <div class="wrapper">
-            <div class="container__block">
-                <p>Contest ID: <a href="{{ route('contests.show', $room->contest_id) }}">{{ $room->contest_id }}</a></p>
-                <p>Campaign ID: {{ $contest->campaign_id }}</p>
-                <p>Campaign Run ID: {{ $contest->campaign_run_id }}</p>
-                <p>Signup Start Date: {{ $room->signup_start_date->format('F d, Y') }}</p>
-                <p>Signup End Date: {{ $room->signup_end_date->format('F d, Y') }}</p>
+            <div class="container__block -half">
+                <h2 class="heading">Information</h2>
 
+                <ul>
+                    <li><strong>Campaign ID:</strong> {{ $contest->campaign_id }}</li>
+                    <li><strong>Campaign Run ID:</strong> {{ $contest->campaign_run_id }}</li>
+                    <li><strong>Contest ID:</strong> <a href="{{ route('contests.show', $room->contest_id) }}">{{ $room->contest_id }}</a></li>
+                    <li><strong>Signup Start Date:</strong> {{ $room->signup_start_date->format('F d, Y') }}</li>
+                    <li><strong>Signup End Date:</strong> {{ $room->signup_end_date->format('F d, Y') }}</li>
+                </ul>
+            </div>
+            <div class="container__block -half">
                 <ul class="form-actions -inline">
+                    <li>
+                        <a href="{{ route('waitingrooms.edit', $room->id) }}" class="button">Edit</a>
+                    </li>
                     @if (has_signup_period_ended($room->signup_end_date))
                         <li>
                             <a href="{{ route('split', $room->id) }}" class="button -secondary">Split room</a>
                         </li>
                     @endif
+                </ul>
+            </div>
 
-                    <li>
-                        <a href="{{ route('waitingrooms.edit', $room->id) }}" class="button">Edit</a>
-                    </li>
-                    <li>
-                        <a href="{{ route('waitingrooms.export', $room->id) }}" class="button">Export</a>
-                    </li>
+        </div>
+    </div>
+
+    <div class="container">
+        <div class="wrapper">
+            <div class="container__block">
+            <h2 class="heading">Data export</h1>
+                <ul class="list">
```

Comment:
Yeah, 1 list item is fine. I'm following how it's set up other places. Could be debated, but I think eventually we're hoping to remove all the exports or minimize their use once emails are working 100%.